### PR TITLE
Public docs update (e109) - VEP and Variation API

### PR DESCRIPTION
**Pull request by nakib103**
The changes can be seen in the sandbox - http://wp-np2-11.ebi.ac.uk:7070/

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html [hunk 1]
```diff
@@ -80,61 +80,64 @@ <h2 id="plugins_existing">Existing plugins</h2>
...
...]]/SpliceRegion.pm">SpliceRegion</a></div></td><td><p>This is a plugin for the Ensembl Variant Effect Predictor (VEP) that<br />provides more granular predictions of splicing effects. ... <a class="button" href="#spliceregion" style="padding:3px 8px 0px 8px !important;height:18px" onclick="show_hide('spliceregion');" id="a_spliceregion">more</a></p><div id="div_spliceregion" style="display:none;"><p>Three additional terms may be added:</p><p># splice_donor_5th_base_variant : variant falls in the 5th base after the splice donor junction (5' end of intron)</p><p>v<br />...EEEEEIIIIIIIIII...</p><p>(E = exon, I = intron, v = variant location)</p><p># splice_donor_region_variant : variant falls in region between 3rd and 6th base after splice junction (5' end of intron)</p><p>vv vvv<br />...EEEEEIIIIIIIIII...</p><p># splice_polypyrimidine_tract_variant : variant falls in polypyrimidine tract at 3' end of intron, between 17 and 3 bases from the end</p><p>vvvvvvvvvvvvvvv<br />...IIIIIIIIIIIIIIIIIIIIEEEEE...</p></div></td><td><div class="vdoc_dtype_count" style="float:left;padding:2px 6px;cursor:default;background-color:#FF0000">Splicing predictions</div></td><td>-</td><td>Ensembl</td></tr>
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:135]: SpliceRegion variant location examples are not properly parsed.

File: docs/htdocs/info/docs/tools/vep/script/vep_plugins.html [hunk 2]
```diff
@@ -80,61 +80,64 @@ <h2 id="plugins_existing">Existing plugins</h2>
...
...L_VERSION]]/ExAC.pm">ExAC</a></div></td><td><p>A VEP plugin that retrieves ExAC allele frequencies. ... <a class="button" href="#exac" style="padding:3px 8px 0px 8px !important;height:18px" onclick="show_hide('exac');" id="a_exac">more</a></p><div id="div_exac" style="display:none;"><p>Visit https://ftp.broadinstitute.org/pub/ExAC_release/current to download the latest ExAC VCF.</p><p>Note that the currently available version of the ExAC data file (0.3) is only available<br />on the GRCh37 assembly; therefore it can only be used with this plugin when using the<br />VEP on GRCh37. See http://www.ensembl.org/info/docs/tools/vep/script/vep_other.html#assembly</p><p>The tabix utility must be installed in your path to use this plugin.</p><p>The plugin takes 3 command line arguments. Second and third arguments are not mandatory. If AC specified as second<br />argument Allele counts per population will be included in output. If AN specified as third argument Allele specific<br />chromosome counts will be included in output.</p></div></td><td><div class="vdoc_dtype_count" style="float:left;padding:2px 6px;cursor:default;background-color:#A9A9A9">ND</div></td><td>-</td><td>Ensembl</td></tr>
```
[the remark this answers is not in the document]
Comment (nakib103) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:97]: We actually needs to omit it from the script. It adds all the plugin from file present (except the ones we told to skip) and not config which I told you before!

[the remark this answers is not in the document]
Comment (nuno-agostinho) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html:97]: Can't we change the script to only get the plugins from config?

[the remark this answers is not in the document]
Comment (nuno-agostinho) [reply docs/htdocs/info/docs/tools/vep/script/vep_plugins.html]: Fixed in https://github.com/Ensembl/ensembl-variation/pull/930 and added updates to script in [ENSVAR-5395](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5395). Thanks!

File: docs/htdocs/info/docs/api/variation/variation_schema.html [hunk 3]
```diff
@@ -707,8 +707,6 @@ <h2>Failed tables</h2>
     <img src="/i/16/plus-button.png" class="sql_schema_icon" alt="show"/> Show species
   </a></p>
     <div id="sp_failed_allele" style="display:none;">      <ul>
-        <li class="sql_schema_species_name">Canis lupus familiaris</li>
-        <li class="sql_schema_species_name">Canis lupus familiarisboxer</li>
```
[the remark this answers is not in the document]
Comment (nuno-agostinho) [reply docs/htdocs/info/docs/api/variation/variation_schema.html:711]: Sounds fair! Thanks!

**Review by nuno-agostinho — APPROVED** [review]
Okay, the major comments have been tackled, thanks @nakib103!

**Pull request merged**